src/app/services/property-detail/property-detail.service.ts

```diff
@@ -1,17 +1,33 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
+import { HttpHeaders, HttpClient } from '@angular/common/http';
 import { HttpService } from 'src/app/services/http.service';
-
+import { APPCONFIG } from 'src/app/config';
+import { LocalStorageService } from 'src/app/services/local-storage.service';
 @Injectable({
     providedIn: 'root'
 })
 export class PropertyDetailService {
+    userToken = this.localStorageService.get('token', '');
+  httpOptions = {
```

Comment:
I think there's an interceptor that adds the Authorization headers to the request, check the `src/app/helpers/jwt.interceptor.ts`, was there a specific reason for having this here?